### DISCUSSION TITLE
20230117 update only single grid view on iPhone

### DISCUSSION THREAD

**Discussion by kw-sudo** [rern/rAudio#1193] [Fixed Issues]
Hi,

Since the latest update, there is only a single cover grid view in library (vertical, basically two in a row).

![8B87028E-128F-493E-B6B4-B63891BC0284](https://user-images.githubusercontent.com/104136223/213183437-b47bb406-c652-462c-9e6e-0c73c1ca216e.jpeg)

Regards,
Oliver

**Comment by kw-sudo** [comment]
On iPad/iMac the grid no longer fills the entire window area, but instead a vertical row is 'missing' at the edge

![safari grid](https://user-images.githubusercontent.com/104136223/213193661-013cef13-f7c5-4a44-928e-a8838b87fdb5.jpg)



Regards,
Oliver

**Reply by Sagnasty** [reply]
I also have this library view behavior

**Comment by kw-sudo** [comment]
@rern : Thank you, it's fixed.
